src/sink/elasticsearch.rs

```diff
@@ -228,13 +228,13 @@ impl Sink<ElasticsearchConfig> for Elasticsearch {
                                 ELASTIC_RECORDS_TOTAL_FAILED
                                     .fetch_add(1, Ordering::Relaxed);
                                 if let Some(cause) = item.cause() {
-                                    error!(
+                                    debug!(
```

Comment:
Why don't we want to know about these in normal production?

Reply:
I suppose the answer is that we already have internal telemetry so the logs are redundant.

Reply:
They're useful for debugging purposes but if you kick them out at an error level _and_ point cernan at itself you get a feedback loop. 

None of these are actionable, really. Just useful information to have for fiddling.